src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -93,7 +93,7 @@
      * But this parameter allows that to be overwritten.
      * </p>
      */
-    @Parameter
+    @Parameter(property = "webAppFile")
```

Comment:
I think using a more qualified name I think `property="hpi.webAppFile` would be slightly better, but not feeling that strongly since anyway the existing code and parameters are not doing this.

Reply:
Yes, I agree, but I've followed the approach of the existing ones.